### PR TITLE
changes the deit and delete icons as well as remove the photogallery …

### PR DESCRIPTION
**Pull request by mehdirajani5**
issue url : https://github.com/AhsanAyaz/e-man/issues/36

reset the edit/ delete icons and change the functionality, when clicked on event row

### CODE REVIEW

File: client/src/app/shared/components/entity-list/entity-list.component.html [hunk 1]
```diff
@@ -34,15 +34,27 @@
...
-        Photos
+
+      <button mat-mini-fab color="warn"
+        class="table-container__row--btn"
+        matTooltip="Delete"
+        matTooltipPosition="above">
+        <mat-icon (click)="openWarningDialog(element,i, $event)">delete</mat-icon>
       </button>
+
+      <!-- <button mat-mini-fab color="basic"
```
Comment (AhsanAyaz) [inline client/src/app/shared/components/entity-list/entity-list.component.html]: Remove unused code

File: client/src/app/shared/components/entity-list/entity-list.component.html [hunk 2]
```diff
@@ -34,15 +34,27 @@
...
+        <mat-icon (click)="openDialog('edit', element, i, $event)">edit</mat-icon>
       </button>
-      <button mat-mini-fab color="basic" *ngIf="entityType == 'event'" (click)="moveToPhotos(element)">
-        Photos
+
+      <button mat-mini-fab color="warn"
+        class="table-container__row--btn"
+        matTooltip="Delete"
+        matTooltipPosition="above">
+        <mat-icon (click)="openWarningDialog(element,i, $event)">delete</mat-icon>
```
Comment (AhsanAyaz) [inline client/src/app/shared/components/entity-list/entity-list.component.html]: Avoid usage of index `i` from the view.
Rename methods

File: client/src/app/shared/components/entity-list/entity-list.component.html [hunk 3]
```diff
@@ -34,15 +34,27 @@
...
     <th mat-header-cell *matHeaderCellDef> Action </th>
     <td mat-cell *matCellDef="let element; let i = index">
-      <button class="table-container__row--btn" mat-mini-fab color="primary" (click)="openDialog('edit', element, i, $event)">
-        Edit
-      </button>
-      <button class="table-container__row--btn" mat-mini-fab color="warn" (click)="openWarningDialog(element,i, $event)">
-        Delete
+
+      <button mat-mini-fab color="primary"
+        class="table-container__row--btn"
```
Comment (AhsanAyaz) [inline client/src/app/shared/components/entity-list/entity-list.component.html]: Usage of proper BEM is required.

Comment (mehdirajani5) [reply client/src/app/shared/components/entity-list/entity-list.component.html]: fixed

**Pull request merged**